fmriprep/workflows/base.py

```diff
@@ -623,6 +623,10 @@ def init_single_subject_wf(subject_id: str):
         num_bold=len(bold_runs)
     )
 
+    # Before initializing BOLD workflow, select/verify anatomical target for coregistration
+    if config.workflow.bold2anat_init in ('auto', 't2w'):
```

Comment:
IMO if someone forces it, they should not get silent fallback.

```suggestion
    if config.workflow.bold2anat_init == 'auto':
```